src/api/queries/useCurrentConfig.ts

```diff
@@ -0,0 +1,24 @@
+import { useOkapiKy } from '@folio/stripes/core';
+import { useQuery } from 'react-query';
+import { SavedJobDTO } from '../dto/types';
+
+export interface CurrentConfigResponse {
+  totalRecords: number;
+  configs?: SavedJobDTO[];
+}
+
+export default function useCurrentConfig() {
+  const ky = useOkapiKy();
+
+  return useQuery<SavedJobDTO | null>(
```

Comment:
I would suggest to follow one naming, in dto you save saved job, hook has current config, it confuses

Reply:
We only use this in the one file, this hook is the current save job.

Reply:
it doesn't matter how often it's used, it's just confusing from names what code does